post_proc/py/plot_scalar_on_native_grid/mpas_plot.py

```diff
@@ -21,6 +22,7 @@
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
 from geopy.distance import distance
+import pandas as pd
```

Comment:
This doesn't seem necessary, right? It seems we are not using Pandas (at least directly) anywhere...

Reply:
Thanks a lot, Felipe. I was planning on using pandas, and then found it to be unnecessary -- but forgot to remove the imports. In reviewing this I realized that also

import cartopy.feature as cfeature
from geopy.distance import distance

were not being used, so I removed them.